bms_factors/temperature.py

```diff
@@ -0,0 +1,23 @@
+class Temperature:
```

Comment:
These three classes repeat the same pattern with different limits and messages. Try another approach by having one class that takes the limits and the messages

Reply:
Done Sir, have created a class to do the exact same part.